catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/ListApplication.java

```diff
@@ -272,7 +294,8 @@ private String contentTypeFromFilename(String filename) {
   private void store(
       AttachmentInfo createInfo,
       Consumer<String> idConsumer,
-      Consumer<String> errorMessageConsumer) {
+      Consumer<String> errorMessageConsumer,
+      Metacard right) {
```

Comment:
Naming the variable right doesn't seem like a descriptive name. How about just calling it metacard?